Casks/r/replay.rb

```diff
@@ -7,10 +7,7 @@
   desc "Time travel debugging"
   homepage "https://replay.io/"
 
-  livecheck do
-    url :url
-    strategy :extract_plist
-  end
+  deprecate! date: "2024-07-24", because: "moved to npm"
```

Comment:
```suggestion
  deprecate! date: "2024-07-24", because: :discontinued
```

It seems like those are two different products: the one available on NPM is the Replay CLI, and the one here is the deprecated [standalone application](https://docs.replay.io/reference/replay-runtimes/replay-firefox) based on Mozilla Firefox.

Reply:
Indeed it does look that way, sorry I missed that.

Reply:
No need to apologize! That's what reviews are for. ❤️